pkg/storage/mongo/cataloger.go

```diff
@@ -37,6 +43,7 @@ func (s *ModuleStore) Catalog(ctx context.Context, token string, pageSize int) (
 	for cursor.Next(ctx) {
 		var module storage.Module
 		if err := cursor.Decode(&module); err != nil {
+			fmt.Println(module)
```

Comment:
One last thing - can you remove the Print here?

Reply:
Sorry, stray statement from debugging 😅

Reply:
wow you're fast :smile: